src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCreateCollectorFileServiceImpl.java

```diff
@@ -109,18 +112,23 @@ protected int calculateBatchSequenceNumber() {
     protected int countCollectorFilesGeneratedFromSAEFilesToday() {
         String wildcardFileName = ConcurConstants.COLLECTOR_CONCUR_OUTPUT_FILE_NAME_PREFIX
                 + KFSConstants.WILDCARD_CHARACTER + GeneralLedgerConstants.BatchFileSystem.EXTENSION;
-        java.sql.Date currentDate = dateTimeService.getCurrentSqlDateMidnight();
-        String currentDateAsString = dateTimeService.toDateString(currentDate);
-        String rangeForCurrentDate = String.format(DATE_RANGE_FORMAT, currentDateAsString, currentDateAsString);
+        String rangeForCurrentDate = buildDateRangeStringForCurrentDate();
         
-        Map<String,String> criteria = new HashMap<>();
-        criteria.put(CUKFSPropertyConstants.PATH, collectorDirectoryPath);
-        criteria.put(KFSPropertyConstants.FILE_NAME, wildcardFileName);
-        criteria.put(CUKFSPropertyConstants.LAST_MODIFIED_DATE, rangeForCurrentDate);
+        MultivaluedMap<String,String> criteria = new MultivaluedHashMap<>();
+        criteria.add(CUKFSPropertyConstants.PATH, collectorDirectoryPath);
+        criteria.add(KFSPropertyConstants.FILE_NAME, wildcardFileName);
+        criteria.add(CUKFSPropertyConstants.LAST_MODIFIED_DATE, rangeForCurrentDate);
         
-        List<? extends BusinessObject> searchResults = batchFileLookupableHelperService.getSearchResults(criteria);
+        List<? extends BusinessObject> searchResults = batchFileLookupableHelperService.getSearchResults(BatchFile.class, criteria);
         return searchResults.size();
     }
+    
+    private String buildDateRangeStringForCurrentDate() {
+        java.sql.Date midnight = dateTimeService.getCurrentSqlDateMidnight();       
+        Date now = new Date(System.currentTimeMillis());
+        String rangeForCurrentDate = String.format(DATE_RANGE_FORMAT, Long.toString(midnight.getTime()), Long.toString(now.getTime()));

```

Comment:
Do these dates have to be specified as long integers instead of date strings? If so, does the upper value need to be changed to the end of the day instead of the current time?